airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e094cb9a-26de-4645-8761-65c0c425d1de
-  dockerImageTag: 3.6.0
+  dockerImageTag: 3.7.0
```

Comment:
This will also have to be updated to 3.9.0 or later depending on the conflicts

Reply:
Updated